akka-http2-support/src/main/scala/akka/http/impl/http2/FrameParser.scala

```diff
@@ -97,7 +107,7 @@ object FrameParser extends ByteStringParser[FrameEvent] {
         ContinuationFrame(streamId, endHeaders, payload.remainingData)
 
       case tpe ⇒ // TODO: remove once all stream types are defined
-        GenericEvent(tpe, flags, streamId, payload.remainingData)
+        UnknownFrameEvent(tpe, flags, streamId, payload.remainingData)
```

Comment:
I think we might need this later as well (or just ignore unknown frames directly).